integration-tests/resteasy-jackson/src/main/resources/application.properties

```diff
@@ -1,2 +1,3 @@
 message=Production
-quarkus.smallrye-openapi.store-schema-directory=target/generated
\ No newline at end of file
+quarkus.smallrye-openapi.store-schema-directory=target/generated
+resteasy.gzip.max.input=5000
```

Comment:
I'm not sure we want to do it this way. My guess is that we would want some proper Quarkus config declared via a proper config class for that.

@stuartwdouglas WDYT?

Reply:
See initial comment .  Resteasy runtime uses the ResteasyConfiguration SPI.

Reply:
@gsmet Also, that was just a test example.  Could have been any value for the test.

Reply:
We do expose this under quarkus.resteasy.gzip.max-input